### ISSUE TITLE
PiecewiseConstantDecay Keras Learning Rate Scheduler not compatible with XLA Compilation

### ISSUE THREAD

**Issue by ecrows** [tensorflow/tensorflow#38055]
**System information**
- TensorFlow version (you are using): 2.1.0
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**

The Keras PieceWiseConstant Decay scheduler currently causes an 'unsupport op' error when used with XLA compilation (i.e. when used with `@tf.function(experimental_compile=True)`).  This happens because [it calls the control_flow_ops `case` function with `exclusive=True`](https://github.com/tensorflow/tensorflow/blob/e3baa0f5b6cb77cbd312f0ad89e46829c073b1ec/tensorflow/python/keras/optimizer_v2/learning_rate_schedule.py#L258), which eventually results in [an assertion that is incompatible with XLA](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/control_flow_ops.py#L3194).

I would like to add an additional `validate_args=True` parameter to PiecewiseConstantDecay that can be optionally set to False to forgo setting `exclusive=True`, thus making the operation TPU-friendly ([similar to the approach taken elsewhere](https://github.com/tensorflow/probability/pull/771)).

**Will this change the current api? How?**

This will not change beahviour of any existing code, but will provide an additional parameter that can be used to leverage XLA compatibility for running on TPU.

**Who will benefit with this feature?**

All users of Cloud TPUs who would like to make complete use of Keras learning rate schedulers.

**Any Other info.**

N/A

**Comment by chunduriv** [comment]
@ecrows, Sorry for the late response. Is this still an issue for you?

Can you please try recent TF2.7 or tf-nightly and let us know whether it is persisting. Also, share a simple standalone code to reproduce the issue. Thanks!
